hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneClientAdapterImpl.java

```diff
@@ -521,8 +521,12 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status,
     OmKeyInfo keyInfo = status.getKeyInfo();
     short replication = (short) keyInfo.getReplicationConfig()
         .getRequiredNodes();
+    //calculate the number of bytes required to store the dataSize with replication
+    long diskConsumed = keyInfo.getReplicatedSize();
```

Comment:
I don't think we need this variable - just use the `keyInfo.getReplicatedSize()` call inline.

Reply:
yes!! definitely!